conf/distro/poky-sota-systemd.conf

```diff
@@ -9,4 +9,4 @@ DISTRO_CODENAME = "sota"
 
 DISTRO_FEATURES_append = " systemd"
 VIRTUAL-RUNTIME_init_manager = "systemd"
-PREFERRED_RPROVIDER_virtual/network-configuration ??= "networkd-dhcp-conf"
+#PREFERRED_RPROVIDER_virtual/network-configuration ??= "networkd-dhcp-conf"
```

Comment:
why?

Reply:
leftover from my experiments, corrected it.